### DISCUSSION TITLE
Learning rate scheduler

### DISCUSSION THREAD

**Discussion by huyduong7101** [huggingface/diffusers#8632] [Q&A]
Learning rate scheduler is an essential technique used in training models to faster the convergence and reduce overfitting. However, as I know so far, learning rate constant is used in almost studies. Are there any reasons behind it?
